src/Components/Inquiry/useInquiry.tsx

```diff
@@ -9,14 +9,19 @@ export interface UseInquiryProps {
 export const useInquiry = ({ artworkID }: UseInquiryProps) => {
   const [isInquiryVisible, setIsInquiryVisible] = useState(false)
   const [askSpecialist, setAskSpecialist] = useState(false)
+  const [enableCreateAlert, setEnableCreateAlert] = useState(false)
```

Comment:
**non-blocking/suggestion:** 

```suggestion
  const [isCreateAlertEnabled, setIsCreateAlertEnabled] = useState(false)
```
You can also ignore it

Reply:
Good idea but I think the consistency of the variable name is valuable here since we're threading it through so many layers so I might leave this as is.